packages/builder/src/components/backend/DataTable/modals/grid/GridCreateViewModal.svelte

```diff
@@ -0,0 +1,60 @@
+<script>
+  import { getContext } from "svelte"
+  import { Input, notifications, ModalContent } from "@budibase/bbui"
+  import { goto } from "@roxi/routify"
+  import { viewsV2 } from "stores/backend"
+
+  const { filter, sort, definition } = getContext("grid")
+
+  let name
+
+  $: views = Object.keys($definition?.views || {})
+  $: nameExists = views.includes(name?.trim())
```

Comment:
I suppose there's no issue with allowing the same name with different casing, since keys in JS objects are case sensitive. I guess we can add in case checking just for consistency since we do it in other places, but I don't think there will be any bugs due to casing. I'll add this in.